apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -3203,7 +3247,8 @@ data class NetCfg(
   val tcpKeepAlive: KeepAliveOpts?,
   val smpPingInterval: Long, // microseconds
   val smpPingCount: Int,
-  val logTLSErrors: Boolean = false
+  val logTLSErrors: Boolean = false,
+  val showSubscriptionPercentage: Boolean = false
```

Comment:
this isn't a part of net config - net config only stores settings for network transport. `showSubscriptionPercentage` is a purely UI setting to show/hide percent.

(NetCfg wouldn't correctly decode/encode as backend doesn't have this field)

```suggestion
```